### PR TITLE
Cats#index

### PR DESCRIPTION
**Pull request by sarkasvo**
index.html.erb for cats created

### CODE REVIEW

File: app/views/cats/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<h1>See below all our cats:</h1>
+
+  <div class="cards">
+
+    <% @cats.each do |cat| %>
+      <div class="card" style="background-image: url(https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/skateboard.jpg)">
+        <!-- path for link to be corrected after show works -->
+        <p><%= link_to cat.name, root_path %></p>
```
Comment (laskyas) [inline app/views/cats/index.html.erb]: add show path

File: app/views/cats/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <div class="cat-info-box">
+              <div class="cat-basic-description"></div>
+              <p>breed: <%= cat.breed %> color: </p>
+              <p> color: <%= cat.color %></p>
+            <!-- <div class="cat-description">
+              <p>description:</p>
+            <%= cat.description %></p>
```
Comment (laskyas) [inline app/views/cats/index.html.erb]: delete

File: app/views/cats/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            </div> -->
+        </div>
+      </div>
+
+    <% end %>
+
+  </div>
+
+<!-- just for testing purposes -->
```
Comment (laskyas) [inline app/views/cats/index.html.erb]: to be removed

**Review by laskyas — CHANGES_REQUESTED** [review]

**Review by patulka — APPROVED** [review]
LGTM :)

**Pull request merged**